cmd/metal-api/internal/datastore/network.go

```diff
@@ -91,39 +91,45 @@ func (p *NetworkSearchQuery) generateTerm(rs *RethinkStore) *r.Term {
 	}
 
 	for _, prefix := range p.Prefixes {
-		ip, length := utils.SplitCIDR(prefix)
+		pfx, err := netip.ParsePrefix(prefix)
+		if err != nil {
+			continue
```

Comment:
could these `continue`s lead to a different behavior? Previously it would set `ip` to be `prefix`. Now it will be ignored.
Or are they forbidden and validated beforehand?

Reply:
We should error out then, will do

Reply:
done

Reply:
Perfect! On line 117 there is another `continue`. Didn't want to open two conversations.